index.html

```diff
@@ -0,0 +1,115 @@
+                <img src="images/hamburger.png" alt="Hamburger Icon">
+            </a>
+        </div>
+    </header>
+
+    <section class="headline">
+        <div class="primarytext">
+            <h1>Hey There. <br>I'm Ezra.</h1>
+        </div>
+
+        <div class="supporting">
+            <h2>I'm a Software Developer</h2>
+        </div>
+
+        <div class="paragraph">
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hesitate to contact me.</p>
+        </div>
+
+        <div class="social">
+            <div>
+                <a href="#">
+                    <img src="images/github.png" alt="Github">
+                </a>
+            </div>
+            <div>
+                <a href="#">
+                    <img src="images/linkedin.png" alt="LinkedIn">
+                </a>
+            </div>
+            <div>
+                <a href="#">
+                    <img src="images/thumbs.png" alt="Thumbs">
+                </a>
+            </div>
+            <div>
+                <a href="#">
+                    <img src="images/twitter.png" alt="Twitter">
+                </a>
+            </div>
+            <div>
+                <a href="#">
+                    <img src="images/M.png" alt="M">
+                </a>
+            </div>
+        </div>
+
+        <div class="vectors">
+            <div class="v-8-1">
+                <img src="images/Vectors/Vector-8-1.png" alt="Vector-8-1">
+            </div>
+
+            <div class="v1">
+                <img src="images/Vectors/Vector-1.png" alt="Vector-1">
+            </div>
+
+            <div class="v2">
+                <img src="images/Vectors/Vector-2.png" alt="Vector-2">
+            </div>
+
+            <div class="v3">
+                <img src="images/Vectors/Vector-3.png" alt="Vector-3">
+            </div>
+
+            <div class="v4">
+                <img src="images/Vectors/Vector-4.png" alt="Vector-4">
+            </div>
+
+            <div class="v5">
+                <img src="images/Vectors/Vector-5.png" alt="Vector-5">
+            </div>
+
+            <div class="v6">
+                <img src="images/Vectors/Vector-6.png" alt="Vector-6">
+            </div>
+
+            <div class="v7">
+                <img src="images/Vectors/Vector-7.png" alt="Vector-7">
+            </div>
+
+            <div class="v-8-2">
+                <img src="images/Vectors/Vector-8-2.png" alt="Vector-8-2">
+            </div>
+
+            <div class="v-8-3">
+                <img src="images/Vectors/Vector-8-3.png" alt="Vector-8-3">
+            </div>
+
+            <div class="v-8-4">
+                <img src="images/Vectors/Vector-8-4.png" alt="Vector-8-4">
+            </div>
+
+            <div class="v-8-5">
+                <img src="images/Vectors/Vector-8-5.png" alt="Vector-8-5">
+            </div>
```

Comment:
- [ ] Your vectors are visually stunning on the page! :+1: :100: However, as per project requirements, it is recommended to add them using CSS instead. This approach is beneficial for several reasons, including improving the website's performance by reducing the page load time, providing better scalability and flexibility, and allowing for easier maintenance and customization of the vectors in the future. Additionally, adding vectors using CSS allows for more seamless integration with other design elements on the page, providing a more cohesive and polished overall look.